Modules/Quotas/test/Quotas.Application.Tests/Tests/Identities/QuotaCreatedForTierIntegrationEventHandlerTests.cs

```diff
@@ -1,18 +1,21 @@
 ﻿using Backbone.Modules.Quotas.Application.Infrastructure.Persistence.Repository;
 using Backbone.Modules.Quotas.Application.IntegrationEvents.Incoming.QuotaCreatedForTier;
 using Backbone.Modules.Quotas.Application.IntegrationEvents.Outgoing;
+using Backbone.Modules.Quotas.Application.Metrics;
+using Backbone.Modules.Quotas.Application.Tests.TestDoubles;
 using Backbone.Modules.Quotas.Domain.Aggregates.Identities;
 using Backbone.Modules.Quotas.Domain.Aggregates.Metrics;
 using Backbone.Modules.Quotas.Domain.Aggregates.Tiers;
 using FakeItEasy;
 using Microsoft.Extensions.Logging;
-using Quotas.Application.Tests.TestDoubles;
 using Xunit;
 
-namespace Quotas.Application.Tests.Tests.Identities;
+namespace Backbone.Modules.Quotas.Application.Tests.Tests.Identities;
 
 public class QuotaCreatedForTierIntegrationEventHandlerTests
 {
+    private IMetricStatusesService _metricStatusesService;
```

Comment:
Why is this a field? It's only used in one test. As I said more than once: only use fields if it's really necessary/if it improves the code quality a lot.

Reply:
So that each test doesn't have to FakeIt and then pass it to the Handler creation while also allowing for tests that want to use it for assertions, setup, etc. to be able to freely do it.

Reply:
You can simply make it optional in the Create method.